### DISCUSSION TITLE
help in achieving a specific issue containing Pivot Tables

### DISCUSSION THREAD

**Discussion by adityaa-codes** [filamentphp/filament#6499] [Help]
Event -> Event Ticket <-> Users
	1 -> M 		M <-> M

i have a relation manager inside event for event tickets

what i want is to create a action button in that relation manager button
which opens a model, and i can assign multiple users to that ticket and quantity
and i want to  have a list of users assigned to that ticket and to edit quantity /detach user from ticket
i wanna do it using modals,  how can i do it  

Thanks in Advance

**Comment by danharrin** [comment]
Please attempt to do it based on the documentation, then ask a specific question if something does not work.

**Reply by adityaa-codes** [reply]
Ok, Thanks Dan
